src/newItinerary/internals/ItineraryItem.style.tsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components'
+
+import { color } from '../../_utils/branding'
+
+export const StyledItineraryItem = styled.li`
+  min-height: 32px;
+
+  > div,
+  > a {
```

Comment:
Not sure about the side-effect if any, but this should work
```suggestion
  > * {